api/TransitionEvent.json

```diff
@@ -101,9 +101,15 @@
             "oculus": "mirror",
             "opera": "mirror",
             "opera_android": "mirror",
-            "safari": {
-              "version_added": "preview"
-            },
+            "safari": [
+              {
+                "version_added": "7"
+              },
+              {
+                "prefix": "WebKit",
+                "version_added": "6"
```

Comment:
```suggestion
                "version_added": "4"
```

Reply:
This is actually not the case!  In Safari 5.1 and older, a transition event had to be created using `document.createEvent('WebKitTransitionEvent')`.

<img width="806" alt="image" src="https://github.com/mdn/browser-compat-data/assets/5179191/bc560bc7-712a-4e2e-be77-43a8897ef100">

Reply:
aha! Thanks for the explanation! 👍